dis_snek/models/discord_objects/user.py

```diff
@@ -465,17 +465,17 @@ async def timeout(
             reason=reason,
         )
 
-    async def kick(self, reason: Absent[str] = MISSING):
+    async def kick(self, reason: Absent[str] = MISSING) -> None:
         """
         Remove a member from the guild.
 
         Args:
             reason: The reason for this removal
 
         """
-        return await self._client.http.remove_guild_member(self._guild_id, self.id)
+        await self._client.http.remove_guild_member(self._guild_id, self.id, reason=reason)
```

Comment:
~~not really a typing fix here~~

Reply:
Technically it is.

Or rather, mypy raises it as a warning when you explicitly return the value of a function that returns None.

Reply:
Oh I was referring to the reason=reason

Reply:
Oh, right, I forgot that I fixed that :D